src/esm_runscripts/filedicts.py

```diff
@@ -0,0 +1,47 @@
+"""
+The file-dictionary implementation
+"""
+
+from collections import UserDict
+
+
+class SimulationFile(UserDict):
+    """
+    Desribes a file used within a ESM Simulation.
+
+    Given a config, you should be able to use this in YAML::
+
+        $ cat dummy_config.yaml
+
+        echam:
+            files:
+                jan_surf:
+                    name_in_pool: T63CORE2_jan_surf.nc
+                    name_in_work: unit.24
+                    filetype: NetCDF
+                    description: >
+                        Initial values used for the simulation, including
+                        properties such as geopotential, temperature, pressure
+
+    And, assuming config is as described above::
+
+        >>> sim_file = SimulationFile(config['echam']['files']['jan_surf'])
+
+    You could then copy the file to the experiment folder::
+
+        >>> sim_file.cp_to_exp_tree()
+    """
+
+    # Do NOT implment an __init__ unless you really think you need to. UserDict
+    # takes care of this in a way is that is considerably smarter than any of us are.
+    #
+    # Please delete my annoying comments before merging into actual release ;-)
+
+    def cp_to_exp_tree(self) -> None:
+        pass
+
+    def ln_to_exp_tree(self) -> None:
+        pass
+
+    def mv_to_exp_tree(self) -> None:
+        pass
```

Comment:
```suggestion
    def cp(self, source, target) -> None:
        pass

    def ln(self, source, target) -> None:
        pass

    def mv(self, source, target) -> None:
        pass
```

Reply:
we need simple `cp`/`mv`/'ln' methods, because we might copy in multiple directions right? Where the direction will be given from the `source` and `target` right?

Reply:
I put this in without `source` and `target` for right now; some things might be extracted from the object's properties. For right now, everything is anyway just a "stub" of a method, actual implementation starts tomorrow.